PlanGrid.Api/PutProject.cs

```diff
@@ -0,0 +1,14 @@
+﻿// <copyright file="PutProject.cs" company="PlanGrid, Inc.">
+//     Copyright (c) 2015 PlanGrid, Inc. All rights reserved.
+// </copyright>
+
+using Newtonsoft.Json;
+
+namespace PlanGrid.Api
+{
+    public class PutProject
```

Comment:
consider more semantic name, for users of lesser abilities? e.g. `ProjectUpdate`